index.bs

```diff
@@ -6554,6 +6577,123 @@ requests will be affected.
 
 </div>
 
+#### The network.setCacheBehavior Command ####  {#command-network-setCacheBehavior}
+
+The <dfn export for=commands>network.setCacheBehavior</dfn> command configures
+the network cache behavior for certain requests.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      network.SetCacheBehavior = (
+        method: "network.setCacheBehavior",
+        params: network.SetCacheBehaviorParameters
+      )
+
+      network.SetCacheBehaviorParameters = {
+        cacheBehavior: "default" / "bypass",
+        ? contexts: [+browsingContext.BrowsingContext]
+      }
+    </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+The <dfn export>WebDriver BiDi cache behavior</dfn> steps given |request| are:
+
+1. Let |context| be null.
+
+1. If |request|'s [=request/window=] is an [=environment settings object=]:
+
+  1. Let |environment settings| be |request|'s [=request/window=]
+
+  1. If there is a [=/browsing context=] whose [=active window=] is |environment
+     settings|' [=environment settings object/global object=], set |context| to
+     the [=top-level browsing context=] for that browsing context.
+
+1. If |context| is not null and [=navigable cache behavior map=] [=set/contains=]
+   |context|, return [=navigable cache behavior map=][|context|].
+
+1. Return [=default cache behavior=].
+
+</div>
+
+<div algorithm="remote end steps for network.setCacheBehavior">
+The [=remote end steps=] given <var ignore>session</var> and |command parameters| are:
+
+1. Let |behavior| be |command parameters|["<code>cacheBehavior</code>"].
+
+1. If |command parameters| does not [=map/contain=] "<code>contexts</code>":
+
+  1. Set the [=default cache behavior=] to |behavior|.
+
+  1. [=map/Clear=] [=navigable cache behavior map=].
```

Comment:
Note: the global cache behavior erases all the specific browsing context's cache behavior previously set.

Reply:
Yes, I think that's the same as the way that enabling an event globally works; if you subscribe for a specific context and then subscribe for all contexts and then unsubscribe for all contexts you don't end up with the event subscription for the original set of contexts.